ouroboros-consensus-shelley-test/test/Test/ThreadNet/RealTPraos.hs

```diff
@@ -75,6 +75,9 @@ minK = 5   -- Less than this increases risk of CP violations
 maxK :: Word64
 maxK = 10   -- More than this wastes execution time
 
+activeSlotCoeff :: Rational
+activeSlotCoeff = 0.5   -- TODO this is high
```

Comment:
I'd like to merge with this TODO in place. This is the same value that was there. It's been unproblematic so far (because the RealTPraos test properties don't check for CP violations -- they are expected with such a high `f`), but I'd like to revisit it later.